src/index.js

```diff
@@ -1,8 +1,17 @@
 import 'regenerator-runtime/runtime' //async/await with Parcel
-import {App} from "./app/App";
+import App from './app/App';
 
 const ONE_SECOND_MILLIS = 1000;
 const SW_API_BASE_URL = process.env.SW_API_BASE_URL || "https://swapi.dev/api";
 const QUIZ_MAX_TIME = process.env.QUIZ_MAX_TIME_SECONDS ? process.env.QUIZ_MAX_TIME_SECONDS * ONE_SECOND_MILLIS : 120 * ONE_SECOND_MILLIS;
 
-window.onload = () => App({options: {swApiBaseUrl: SW_API_BASE_URL, quizMaxTime: QUIZ_MAX_TIME}})
+window.onload = () => new App({ options: { swApiBaseUrl: SW_API_BASE_URL, quizMaxTime: QUIZ_MAX_TIME } });
+
+const { frame } = require("src/app/lightsaber.js");
```

Comment:
you should add frame and countdown in App.js file like timer component (btw timer is only a template and isn't necesary here :) )